src/main/java/io/github/jhipster/config/JHipsterProperties.java

```diff
@@ -236,6 +242,81 @@ public void setMaxEntries(long maxEntries) {
                 this.maxEntries = maxEntries;
             }
         }
+
+        public static class Infinispan {
+
+            private String configFile = "default-configs/default-jgroups-tcp.xml";
+
+            private final Local local = new Local();
+
+            private final Distributed distributed = new Distributed();
+
+            private final Replicated replicated = new Replicated();
+
+            public String getConfigFile() { return configFile; }
```

Comment:
@srinivasa-vasu could you plz format the new code similar to remaining of the file so that getter/setters arent on the same line to be consistent with codestyle of the file